packages/x-tree-view/src/internals/plugins/useTreeViewItems/useTreeViewItems.types.ts

```diff
@@ -105,11 +105,17 @@ export interface UseTreeViewItemsParameters<R extends {}> {
    * @default (item) => item.id
    */
   getItemId?: (item: R) => TreeViewItemId;
+  /**
+   * Indentation in pixels between an item and its children.
```

Comment:
May6be it is more specific this way?
```suggestion
   * Horizontal indentation of an item.
```

Reply:
I think we need to keep the information saying that it's the indentation between two level of depths, not the indentation of every items

Reply:
Sounds good to me 😄